src/main/java/techreborn/packets/ServerboundPackets.java

```diff
@@ -171,6 +174,29 @@ public static void init() {
 				}
 			});
 		}));
+
+		NetworkManager.registerServerBoundHandler(LAUNCH_SPEED, ((server, player, handler, buf, responseSender) -> {
+			BlockPos pos = buf.readBlockPos();
+			int buttonAmount = buf.readInt();
+
+			server.execute(() -> {
+				BlockEntity blockEntity = player.world.getBlockEntity(pos);
+				if (blockEntity instanceof LaunchpadBlockEntity) {
+					((LaunchpadBlockEntity) blockEntity).handleGuiInputFromClient(buttonAmount);
+				}
+			});
+		}));
+
+		NetworkManager.registerServerBoundHandler(new Identifier("reborncore", "jump"), (server, player, handler, buf, responseSender) -> {
```

Comment:
You shouldnt listen to other mods packets like this. The packet shoudl be sent from TR and recieved by TR

Reply:
But the sending happens in the mixin? How am i supposed to have the mixin in core and the sending in TR?

Reply:
You should create and event in TR, RC should not be sending jump packets.

Reply:
I'm out of my depth at this point then.